README.md

```diff
@@ -10,7 +10,7 @@ Feature flippers.
 ## MAKE SURE TO READ THIS: 2.X Changes and Migration Path
 
 As of rollout-2.x, only one key is used per feature for performance reasons.
-The data format is `percentage|user_id,user_id,...|group,_group...`. This has
+The serialized format is `percentage|user_id,user_id,...|group,_group...|data_key_1=>data_val_1|data_key_2=>data_val_2|...`. This has
```

Comment:
No, I just didn't think of it, but I like that idea. I updated the pull request to use JSON (it's much cleaner, yay!)